src/docs/release/breaking-changes/eliminating-nullok-parameters.md

```diff
@@ -0,0 +1,266 @@
+([`InheritedWidget`]s) using static member functions that are typically called
+`of`, and take a `BuildContext`.
+
+Before non-nullability was the default, it was useful to have a toggle on these
+APIs that swapped between throwing an exception if the widget was not present in
+the widget tree and returning null if it was not found. It was useful, and
+wasn't confusing, since every variable was nullable.
+
+When non-nullability was made the default, it was then desirable to have the
+most commonly used APIs return a non-nullable value. This is because saying
+`MediaQuery.of(context, nullOk: false)` and then still requiring an `!` operator
+or `?` and a fallback value after that call felt awkward.
+
+The `nullOk` parameter was a cheap form of providing a null safety toggle, which
+in the face of true language support for non-nullability, was then supplying
+redundant, and perhaps contradictory signals to the developer.
+
+To solve this, the `of` accessors (and some related accessors that also used
+`nullOk`) were split into two calls: one that returned a non-nullable value and
+threw an exception when the sought-after widget was not present, and one that
+returned a nullable value that didn't throw an exception, and returned null if
+the widget was not present.
+
+The design document for this change is here:
+[https://flutter.dev/go/eliminating-nullok-parameters](
+https://flutter.dev/go/eliminating-nullok-parameters)
+
+## Description of change
+
+The actual change modified these APIs to not have a `nullOk` parameter, and to
+return a non-nullable value:
+
+1. [`MediaQuery.of`]
+1. [`Navigator.of`]
+1. [`ScaffoldMessenger.of`]
+1. [`Scaffold.of`]
+1. [`Router.of`]
+1. [`Localizations.localeOf`]
+1. [`FocusTraversalOrder.of`]
+1. [`FocusTraversalGroup.of`]
+1. [`Focus.of`]
+1. [`Shortcuts.of`]
+1. [`Actions.handler`]
+1. [`Actions.find`]
+1. [`AnimatedList.of`]
+1. [`SliverAnimatedList.of`]
+1. [`CupertinoDynamicColor.resolve`]
+1. [`CupertinoDynamicColor.resolveFrom`]
+1. [`CupertinoUserInterfaceLevel.of`]
+1. [`CupertinoTheme.brightnessOf`]
+1. [`CupertinoThemeData.resolveFrom`]
+1. [`NoDefaultCupertinoThemeData.resolveFrom`]
+1. [`CupertinoTextThemeData.resolveFrom`]
+1. [`MaterialBasedCupertinoThemeData.resolveFrom`]
+
+And introduced these new APIs alongside those, to return a nullable
+value:
+
+1. [`MediaQuery.maybeOf`]
+1. [`Navigator.maybeOf`]
+1. [`ScaffoldMessenger.maybeOf`]
+1. [`Scaffold.maybeOf`]
+1. [`Router.maybeOf`]
+1. [`Localizations.maybeLocaleOf`]
+1. [`FocusTraversalOrder.maybeOf`]
+1. [`FocusTraversalGroup.maybeOf`]
+1. [`Focus.maybeOf`]
+1. [`Shortcuts.maybeOf`]
+1. [`Actions.maybeHandler`]
+1. [`Actions.maybeFind`]
+1. [`Actions.maybeInvoke`]
+1. [`AnimatedList.maybeOf`]
+1. [`SliverAnimatedList.maybeOf`]
+1. [`CupertinoDynamicColor.maybeResolve`]
+1. [`CupertinoDynamicColor.maybeResolveFrom`]
+1. [`CupertinoUserInterfaceLevel.maybeOf`]
+1. [`CupertinoTheme.maybeBrightnessOf`]
+1. [`CupertinoThemeData.maybeResolveFrom`]
```

Comment:
I have not been able to find this method on master.

Reply:
This one may be missing.

Reply:
Yep.